packages/video_player_videohole/example/lib/main.dart

```diff
@@ -11,6 +11,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:http/http.dart' as http;
 import 'package:video_player_videohole/video_player.dart';
+import 'package:video_player_videohole/video_player_platform_interface.dart';
```

Comment:
It doesn't seem a good idea to make all the types in `video_player_platform_interface.dart` available to the users. You can use the `show` keyword to export specific types in the library. It would be still better to move the code to `video_player.dart` or a new file.

Reply:
`Track` class can't be moved to `video_player.dart`, because `video_player_platform_interface.dart` and `video_player_tizen.dart` must use these classes.
Maybe it's better to create a new file and import it.

Reply:
Fixed.